packages/common/src/ui/layout/tables/components/Cells/CheckboxCell/CheckboxCell.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { CellProps } from '../../../columns';
+import { Checkbox } from '@common/components';
+import { RecordWithId } from '@common/types';
+import { useBufferState } from '@common/hooks';
+
+export const CheckboxCell = <T extends RecordWithId>({
+  rowData,
+  column,
+  rows,
+}: CellProps<T>): React.ReactElement<CellProps<T>> => {
+  const [buffer, setBuffer] = useBufferState(
+    column.accessor({ rowData, rows })
+  );
+
+  return (
+    <Checkbox
+      checked={!!buffer}
+      size="small"
+      onChange={e => {
+        const newValue = e.target.checked;
+        setBuffer(newValue);
+        column.setter({ ...rowData, [column.key]: newValue });
+      }}
+    />
+  );
+};
```

Comment:
Oh. Yes, I also changed the stocktake edit modal to use the batch/checkbox columns in the data table which fixes the tabbing problem which other modals suffered from.. that is what prompted me to do a lot of this shuffling around..